roles/pulp_webserver/defaults/main.yml

```diff
@@ -6,3 +6,7 @@ pulp_api_host: 127.0.0.1
 pulp_api_port: 24817
 pulp_configure_firewall: auto
 pulp_user_home: '/var/lib/pulp'
+
+pulp_webserver_disable_https: false
+pulp_webserver_tls_folder: '{{ pulp_config_dir }}'
+pulp_webserver_httpd_servername: '{{ ansible_nodename }}'
```

Comment:
`playbooks/example-use/group_vars/all` has:
```
content_origin: "http://{{ ansible_fqdn }}"
```
We had a discussion at the time on which hostname-like variable to use by default, and decided on that, because any SSL cert must be an FQDN. We were thinking forward to this :)